packages/hardhat/test/fixture-projects/no-anvil-config/hardhat.config.ts

```diff
@@ -0,0 +1,11 @@
+// We load the plugin here.
+import { HardhatUserConfig } from "hardhat/types";
+
+import "../../../src/index";
+
+const config: HardhatUserConfig = {
+  solidity: "0.7.3",
+  defaultNetwork: "anvil"
```

Comment:
can we not require this at all? have the hardhat plugin do `hre.config.defaultNetwork = "anvil"` for the user?